src/Modules/Windows-Core/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1

```diff
@@ -24,8 +24,8 @@ CmdletsToExport= "Format-List", "Format-Custom", "Format-Table", "Format-Wide",
     "Unblock-File", "Get-Runspace", "Debug-Runspace", "Enable-RunspaceDebug", "Disable-RunspaceDebug",
     "Get-RunspaceDebug", "Wait-Debugger" 
 FunctionsToExport= "Get-FileHash", "New-TemporaryFile", "New-Guid", "Format-Hex", "Import-PowerShellDataFile",
-    "ConvertFrom-SddlString"
-AliasesToExport= "fhx"
+    "ConvertFrom-SddlString", "Get-Uptime"
+AliasesToExport= "fhx", "uptime"

```

Comment:
Haven't we been through this before with ls and curl i.e. superseding a native utility with a PowerShell alias.  I wouldn't define this alias on *nix.  Windows perhaps but not *nix.